src/components/LocationSelector/LocationSelector.tsx

```diff
@@ -141,6 +148,20 @@ export const LocationSelector: React.FC<Props> = (props) => {
         }
     }
 
+    function handleButtonKeyPress(e: React.KeyboardEvent<HTMLButtonElement>) {
+        if (e.key === ESCAPE_KEY) {
+            buttonRef.current?.blur();
+        } else if (!(e.key === TAB_KEY || e.key === ESCAPE_KEY)) {
```

Comment:
I spent some time to figuring out what this condition exactly is for 😬
What about replacing it with (e.key !== TAB_KEY && e.key !== ESCAPE_KEY)? I think it looks a little more familiar but maybe it's just me.